packages/manager/apps/web/client/app/hosting/cdn/shared/hosting-cdn-shared-settings.routing.js

```diff
@@ -0,0 +1,69 @@
+import includes from 'lodash/includes';
+
+import template from './leave-confirm-settings/leave/hosting-shared-leave-settings.html';
+import controller from './leave-confirm-settings/leave/hosting-shared-leave-settings.controller';
+import CdnSharedSettingsController from './hosting-cdn-shared-settings.controller';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('app.hosting.cdn.shared', {
+    url: '/shared/settings/:domainName',
+    params: {
+      domain: null,
+    },
+    component: 'hostingCdnSharedSettings',
+    resolve: {
+      serviceName: /* @ngInject */ ($transition$) =>
```

Comment:
Not already injected in parent state (`app.hosting)` ?

Reply:
Yes right, fixed